### ISSUE TITLE
-Wuninitialized in drivers/gpu/drm/vmwgfx/vmwgfx_stdu.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1811]
```
drivers/gpu/drm/vmwgfx/vmwgfx_stdu.c:509:29: error: variable 'dst_pitch' is uninitialized when used here [-Werror,-Wuninitialized]
        src_offset = ddirty->top * dst_pitch + ddirty->left * stdu->cpp;
                                   ^~~~~~~~~
drivers/gpu/drm/vmwgfx/vmwgfx_stdu.c:492:26: note: initialize the variable 'dst_pitch' to silence this warning
        s32 src_pitch, dst_pitch;
                                ^
                                 = 0
1 error generated.
```

Patch submitted: https://lore.kernel.org/20230314-vmware-wuninitialized-v1-1-1bb4b0989758@kernel.org/

**Comment by nathanchance** [comment]
Correct patch submitted: https://lore.kernel.org/20230314211445.1363828-1-zack@kde.org/

**Comment by nathanchance** [comment]
The fix is now in next-20230316.
